### PR TITLE
updating Dockerfile.rocm to use newer versions of rocblas+miopen which are required for BnA fusion

### PR DESCRIPTION
**Pull request by deven-amd**

### CODE REVIEW

**Comment by sunway513** [comment]
Thanks @deven-amd , LGTM

File: tensorflow/tools/ci_build/Dockerfile.rocm [hunk 1]
```diff
@@ -64,9 +64,12 @@ RUN apt-get update --allow-insecure-repositories && \
     rm -rf /var/lib/apt/lists/*
 
 
-# Override the MIOpen install with a newer version that has fusion API fixes/updates
+# Override the default rocBLAS and MIOpen installs with a newer version with support for BnA fusion
+RUN mkdir -p $HOME/pkgs/rocblas
+RUN cd $HOME/pkgs/rocblas && wget https://www.dropbox.com/s/dvjmyoeyoiotc5s/rocblas-0.14.3.0-Linux.deb && dpkg -i *.deb
```
Comment (whchung) [inline tensorflow/tools/ci_build/Dockerfile.rocm]: for 0.14.3.0, can we use official release package on github?

https://github.com/ROCmSoftwarePlatform/rocBLAS/releases

Comment (sunway513) [reply tensorflow/tools/ci_build/Dockerfile.rocm]: Yes, that'll also work.

Comment (deven-amd) [reply tensorflow/tools/ci_build/Dockerfile.rocm]: Yes that is better...will make the change once CI completes

**Comment by whchung** [comment]
@deven-amd Dont worry about failures on CUDA path. I've fixed it in the other PR #228 . ROCm path looks fine thus far. Please fix rocblas installation path and I'll take this PR.

**Comment by deven-amd** [comment]
> @deven-amd Dont worry about failures on CUDA path. I've fixed it in the other PR #228 . ROCm path looks fine thus far. Please fix rocblas installation path and I'll take this PR.

done. committed the change, re-based it, and pushed out.

**Pull request merged**